automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -1,26 +1,30 @@
-$ErrorActionPreference = 'Stop'
-
-$chromium_string = "\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Chromium"
-$hive = "hkcu"
-$Chromium = $hive + ":" + $chromium_string
-
-if (Test-Path $Chromium) {
-  $silentArgs = '--do-not-launch-chrome'
-} else {
-  $silentArgs = '--system-level --do-not-launch-chrome'
-}
-
-
-$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-
-$packageArgs = @{
-  packageName   = 'chromium'
-  file          = "$toolsdir\chromium-sync_x32.exe"
-  file64        = "$toolsdir\chromium-sync_x64.exe"
-  fileType      = 'exe'
-  silentArgs    = $silentArgs
-  validExitCodes= @(0)
-  softwareName  = 'Chromium'
-}
-
-Install-ChocolateyInstallPackage @packageArgs
+﻿$ErrorActionPreference = 'Stop'
+$scriptDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
+. (Join-Path $scriptDir 'helper.ps1')
+
+$chromium_string = "\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Chromium"
+$hive = "hkcu"
+$Chromium = $hive + ":" + $chromium_string
+
+if (Test-Path $Chromium) {
+  $silentArgs = '--do-not-launch-chrome'
+} else {
+  $silentArgs = '--system-level --do-not-launch-chrome'
+}
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$version = "63.0.3239.108"
+Get-CompareVersion -version $version -notation "-snapshots" -package "chromium"

```

Comment:
What is the purpose of this.
As far as I can see, the only purpose is to write out a message, but nothing else.
Did you mean to have this one behave a little differently?
Like skipping installation if it's already installed?

Reply:
Yes, that is the intention. I will change it to set a bool for the install as a catch.